libs/sdk-backend-spi/src/workspace/execution/export.ts

```diff
@@ -1,5 +1,7 @@
 // (C) 2019-2024 GoodData Corporation
 
+import { IFilter } from "@gooddata/sdk-model";
+
 /**
  * Configuration for exports of results into XLSX or CSV.
```

Comment:
@dremileta just noticed this is not up to date anymore either, let's adjust it in a way we don't have to change it with each new format, sth like this:

`Configuration for exports of results into tabular formats`